src/components/Nft/NftViewer.tsx

```diff
@@ -17,6 +17,7 @@ import { BigNumber } from "bignumber.js";
 import { useSelector } from "react-redux";
 import { Button, Icons } from "@ledgerhq/native-ui";
 import { useTranslation, Trans } from "react-i18next";
+import Clipboard from "@react-native-community/clipboard";
```

Comment:
I'm wondering how it could build before without that import :thinking:

Reply:
We don't have any CI anymore on TS files... So that + everything is red in VSCode cause we don't have either the proper configuration, or we didn't take the time to actually fix all of our errors, it makes it almost impossible to catch this kind of things...